fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -2477,12 +2478,12 @@ private boolean areAllRecordTypesSince(@Nullable Collection<RecordType> recordTy
         }
         final boolean metaDataVersionChanged = oldMetaDataVersion != newMetaDataVersion;
```

Comment:
It looks like `oldMetaDataVersion`, in this context, is equal to 0 on uninitialized stores rather than -1.

Reply:
Yes. It turns out not to matter if `metaDataVersionChanged` is `false` in the logging because `newStore` takes precedence. However, I agree that it would still be superior to always use the same idiom so that it's straightforwardly true and doesn't break in the face of future changes.